### PR TITLE
HTCONDOR-2097 merge sections under cm section

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/admin-manual/ap-policy-configuration.rst [hunk 1]
```diff
@@ -199,3 +199,144 @@ requirements list. Currently, only one (the most recent) problem is
...
+point):
+
+.. code-block:: condor-config
+
+    MASTER_HA_LIST = SCHEDD
+    SPOOL = /share/spool
+    HA_LOCK_URL = file:/share/spool
+    VALID_SPOOL_FILES = $(VALID_SPOOL_FILES) SCHEDD.lock
+
+Configuration macro ``MASTER_HA_LIST`` :index:`MASTER_HA_LIST`
```
Comment (ColeBollig) [inline docs/admin-manual/ap-policy-configuration.rst:252]:
```suggestion
Configuration macro :macro:`MASTER_HA_LIST`
```

File: docs/admin-manual/ap-policy-configuration.rst [hunk 2]
```diff
@@ -199,3 +199,144 @@ requirements list. Currently, only one (the most recent) problem is
...
+watched to make sure that it is running. Each machine with this
+configuration must have access to the lock (the job queue) which
+synchronizes which single machine does run the *condor_schedd* daemon.
+This lock and the job queue must both be located in a shared file space,
+and is currently specified only with a file URL. The configuration
+specifies the shared space (``SPOOL``), and the URL of the lock.
+*condor_preen* is not currently aware of the lock file and will delete
+it if it is placed in the ``SPOOL`` directory, so be sure to add file
+``SCHEDD.lock`` to ``VALID_SPOOL_FILES``
+:index:`VALID_SPOOL_FILES`.
```
Comment (ColeBollig) [inline docs/admin-manual/ap-policy-configuration.rst:263]:
```suggestion
``SCHEDD.lock`` to :macro:`VALID_SPOOL_FILES`.
```

File: docs/admin-manual/ap-policy-configuration.rst [hunk 3]
```diff
@@ -199,3 +199,144 @@ requirements list. Currently, only one (the most recent) problem is
...
+.. code-block:: console
+
+      $ condor_submit -remote condor@example.com myjob.submit
+
+This implies the identification of a single *condor_schedd* daemon,
+running on a single machine. With the high availability of the job
+queue, there are multiple *condor_schedd* daemons, of which only one at
+a time is acting as the single submission point. To make remote
+submission of jobs work properly, set the configuration variable
+``SCHEDD_NAME`` :index:`SCHEDD_NAME` in the local configuration to
```
Comment (ColeBollig) [inline docs/admin-manual/ap-policy-configuration.rst:308]:
```suggestion
:macro:`SCHEDD_NAME` in the local configuration to
```

File: docs/admin-manual/cm-configuration.rst [hunk 4]
```diff
@@ -833,4 +940,819 @@ Running with multiple negotiators also means you need to be careful with the
...
+primary negotiator is unable to run due to misconfiguration. In these
+cases, the *condor_master* will kill the *condor_had* after the
+*condor_negotiator* exists, wait a short period, then restart
+*condor_had*. The *condor_had* will then win the election, so the
+secondary *condor_negotiator* will be killed, and the primary will be
+restarted, only to exit again. If this happens too quickly, neither
+*condor_negotiator* will run long enough to complete a negotiation
+cycle, resulting in no jobs getting started. Increasing this value via
+``MASTER_HAD_BACKOFF_CONSTANT``
+:index:`MASTER_HAD_BACKOFF_CONSTANT` to be larger than a typical
```
Comment (ColeBollig) [inline docs/admin-manual/cm-configuration.rst:1115]:
```suggestion
:macro:`MASTER_HAD_BACKOFF_CONSTANT` to be larger than a typical
```

File: docs/admin-manual/cm-configuration.rst [hunk 5]
```diff
@@ -833,4 +940,819 @@ Running with multiple negotiators also means you need to be careful with the
...
+negotiation cycle can help solve this problem.
+
+To run a high availability pool without the replication feature, do the
+following operations:
+
+#. Set the ``HAD_USE_REPLICATION`` :index:`HAD_USE_REPLICATION`
```
Comment (ColeBollig) [inline docs/admin-manual/cm-configuration.rst:1121]:
```suggestion
#. Set the :macro:`HAD_USE_REPLICATION`
```

File: docs/admin-manual/cm-configuration.rst [hunk 6]
```diff
@@ -833,4 +940,819 @@ Running with multiple negotiators also means you need to be careful with the
...
+invalidated. It may be useful to place an invalidated ClassAd in the absent
+state, instead of having it removed as an invalidated ClassAd. An example of a
+ClassAd that could benefit from being absent is a system with an
+uninterruptible power supply that shuts down cleanly but unexpectedly as a
+result of a power outage. To cause all invalidated ClassAds to become absent
+instead of invalidated, set :macro:`EXPIRE_INVALIDATED_ADS` to ``True``.
+Invalidated ClassAds will instead be treated as if they expired, including when
+evaluating :macro:`ABSENT_REQUIREMENTS`.
+
+GPUs
```
Comment (ColeBollig) [inline docs/admin-manual/cm-configuration.rst:1604]: Should this GPU section be in the cm-configuration and not the ep-configuration?

**Comment by ColeBollig** [comment]
Ran a clean local build and got the following:

/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/ap-policy-configuration.rst:341: WARNING: undefined label: 'admin-manual/ep-configuration:startd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:67: WARNING: unknown document: '/admin-manual/high-availability'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:1790: WARNING: unknown document: '/admin-manual/high-availability'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:7068: WARNING: undefined label: 'admin-manual/monitoring:absent classads'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:7095: WARNING: undefined label: 'admin-manual/monitoring:absent classads'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:10609: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:10784: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-macros.rst:10930: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:121: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:166: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:172: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:178: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:184: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:190: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:196: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:202: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:208: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:214: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'
/Users/colebollig/Desktop/condor_source/master/docs/admin-manual/configuration-templates.rst:226: WARNING: undefined label: 'admin-manual/monitoring:elasticsearch'
/Users/colebollig/Desktop/condor_source/master/docs/getting-htcondor/admin-quick-start.rst:206: WARNING: unknown document: '../admin-manual/monitoring'
/Users/colebollig/Desktop/condor_source/master/docs/version-history/feature-versions-10-x.rst:897: WARNING: undefined label: 'admin-manual/daemon-cron:startd cron and schedd cron'

File: docs/admin-manual/cm-configuration.rst (hunk 6 above)
Comment (GregThain) [reply docs/admin-manual/cm-configuration.rst:1604]: I went back and forth on that...

Comment (ColeBollig) [reply docs/admin-manual/cm-configuration.rst:1603]: Decided to leave GPU section in the CM configuration page. If needed we can move this to the EP configuration page or add a link to this section

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**